tex/artikel/Lehramt.tex

```diff
@@ -1,18 +1,18 @@
+Innerhalb des Studiums werden Vorlesungen, Übungen und Seminare in Modulen zusammen gefasst, die dann in Modulabschlussprüfungen abgeschlossen werden. In den ersten drei Semestern finden die Modulabschlussprüfungen als schriftliche Klausuren statt, danach sind es vor allem mündliche Prüfungen. Die Noten dieser Prüfungen gehen alle in eure spätere Endnote mit ein. Eine Ausnahme stellen die Klausuren in Physik 1 und 2 dar, hier zählt nur die bessere der beiden Noten. Voraussetzung ist dabei selbstverständlich das Bestehen beider Module. Außerdem ist es leider nicht möglich die Modulabschlussprüfungen beliebig oft zu wiederholen, deswegen gibt es in den Klausuren in Physik 1 bis 3 vier Versuche und in allen weiteren nur drei.
 \subsection*{Die Grundmodule}
-In den ersten drei Semestern belegt ihr die grundlegenden Module für das Physikstudium: Dynamik der Teilchen und Teilchensysteme (Physik 1), Thermodynamik und Elektromagnetismus (Physik 2) und Wellen und Quanten (Physik 3). Physik 1 deckt sich im Prinzip mit dem Modul der 1-Fach-Bachelor, nur dass ihr noch ein spezielles Mathetutorium hört.In Physik 2 und Physik 3 hören die 1-Fach-Bachelor zusätzlich noch Theoretische Ergänzungen (TE). Zu den oben genannten jeweiligen Vorlesungen gibt es außerdem noch Übungen in denen ihn Aufgaben zu den Vorlesungen bekommt.\\ Im dritten und vierten Semester muss außerdem noch das Modul Experimentelle Übungen absolviert werden. Innerhalb dieses Moduls werdet ihr alle zwei Wochen eigene Experimente durchführen und dazu Versuchsberichte schreiben die bewertet werden.
+In den ersten drei Semestern belegt ihr die grundlegenden Module für das Physikstudium: Dynamik der Teilchen und Teilchensysteme (Physik 1), Thermodynamik und Elektromagnetismus (Physik 2) und Wellen und Quanten (Physik 3). Die Module decken sich im Prinzip mit denen der 1-Fach-Bachelor (1FB) und bestehen aus drei Vorlesungsterminen und zwei (für Physik 1) bzw. einem (Physik 2 \& 3) Übungstermin pro Woche. In der Übung müsst ihr wöchentlich Aufgaben zum Stoff der Vorlesung lösen und insgesamt die Hälfte der Punkte erreichen, um zur abschließenden Klausur zugelassen zu werden. Die Klausur orientiert sich dann ebenfalls an den Inhalten der Übungsaufgaben. Im Unterschied zum 1FB gibt es für euch in Physik 1 noch ein spezielles Mathetutorium. Dieses besteht aus einem verpflichtenden, aber unbenoteten, Test zu Beginn des Semesters und einem wöchentlichen, freiwilligen Tutorium, welches allerdings sehr hilfreich für die Übungen ist. In Physik 2 und Physik 3 hören die 1-Fach-Bachelor zusätzlich noch Theoretische Ergänzungen (TE), die ihr nicht belegen müsst.\\ Im dritten und vierten Semester muss außerdem noch das Modul Experimentelle Übungen, manchmal auch Grundpraktikum genannt, absolviert werden. Innerhalb dieses Moduls werdet ihr alle zwei Wochen eigene Experimente durchführen und diese in Versuchsberichten auswerten, die bewertet werden und eure Modulnote ergeben.
```

Comment:
Das war vorher auch schon so, aber in jedem Fall: Grundsätzlich wäre es glaube ich sinnvoll, neue Zeilen wirklich in neue Zeilen zu packen, also:
```
abc\\
def
```
Oder man macht natürlich gleich einen neuen Absatz:
```
abc

def
```
In einer tex-Zeile neue Zeilen unterzubringen liest sich finde ich etwas verwirrend. Ist aber natürlich nicht schlimm.